bigcode_eval/tasks/humaneval.py

```diff
@@ -83,9 +122,14 @@ def postprocess_generation(self, generation, idx):
             index of doc in the dataset to which the generation belongs
             (not used for Humaneval-Task)
         """
+        # Want to remove one shot example and any context when post processing
         prompt = self.get_prompt(self.dataset["test"][idx])
```

Comment:
Can we delete this line if it's no longer used?

Reply:
I think we will need to keep this as we need the length of the prompt to isolate the generation in the next line.

Reply:
Got it, missed that usage!